src/edges_cal/alanmode.py

```diff
@@ -363,6 +364,15 @@ def _average_spectra(
     return spfreq, spectra
 
 
+def get_s11date(datadir, year, day):
+    """Return the name of the s11 date: 2022_318_14 for an input (2022,316)."""
```

Comment:
Does this function return the *closest* day if the day given doesn't exist? If so, please add this to the docstring